db/blip_handler.go

```diff
@@ -562,10 +558,15 @@ func (bh *blipHandler) buildChangesRow(change *ChangeEntry, changeVersion Change
 	var changeRow []interface{}
 
 	// change map should only have one entry
-	revs := slices.Collect(maps.Values(changeVersion))
-	if len(revs) > 1 {
+	if len(changeVersion) > 1 {
 		base.AssertfCtx(bh.loggingCtx, "more changes in list than expected on change entry: %v", change.ID)
 	}
+	var rev string
+	if revID, ok := changeVersion[ChangesVersionTypeRevTreeID]; ok {
+		rev = revID
+	} else {
+		rev = changeVersion[ChangesVersionTypeCV]
+	}
```

Comment:
Nit: I think we should have a helper method on `ChangeByVersionType` to return this `rev` string value.

I can see other areas of code (`DocIDChangesFeed`->`createChangesEntry`) that _should_ be using this logic but aren't (filed CBG-4836 to correct that - but will be easier if this code becomes a method and is available to use)